__Data Provided:__
•	The theoretical weekly operating cost model: W = $1,000 + $5X
•	Sample of 25 restaurants with a mean weekly cost of Rs. 3,050
•	Number of units produced in a week (X) follows a normal distribution with a mean (μ) of 600 units and a standard deviation (σ) of 25 units


__1. State the Hypotheses statement:__

H0 = The cost model is accurate, it follows the equation W = $1,000 + $5X

H1 = The cost model is not accurate, it does't follow the current equation.

____2. Calculate the Test Statistic:____

In [5]:
import math
sample_mean = 3050
assumed_mean = 1000 + 5 * 600
std = 125
n = 25

In [6]:
t_test = (sample_mean - assumed_mean)/(std/math.sqrt(n))
t_test

-38.0

__3. Determine the Critical Value:__

alpha = 0.05 

z score = -1.645 (lower tail test)

since the t test value is less than the z score we reject the null hupothesis(H0) and conclude that the current model is inaccurate and produces inefficient results. 

__CONCLUSION__

There is a strong evidence that proves the restaurant owners claims as the t test value -38 is very less than the critical value of -1.64. 

This means that the Bombay hospitality Ltd. should reconside their operating cost model. 

The claims of the restaurant owners seems to be true and we conclude that the cost model is not accurate and it does't follow the current equation.

__CHI SQUARE TEST__

*1. State the Hypotheses:*

   H0 = there's a significant association between the type of smart home device purchased (Smart Thermostats vs. Smart Lights) and the customer satisfaction level.
   
   H1 = there's no  significant association between the type of smart home device purchased (Smart Thermostats vs. Smart Lights) and the customer satisfaction level.

__2. Compute the Chi-Square Statistic:__

In [3]:
import pandas as pd

In [11]:
data = pd.read_csv(r"C:\Users\KIIT\Downloads\data1.csv")
data

Satisfaction  SmartThermostat  SmartLight  Total  \
0    VerySatisfied             50.0        70.0  120.0   
1        Satisfied             80.0       100.0  180.0   
2          Neutral             60.0        90.0  150.0   
3      Unsatisfied             30.0        50.0   80.0   
4  VeryUnsatisfied             20.0        50.0   70.0   
5            Total            240.0       360.0  600.0   
6              NaN              NaN         NaN    NaN   

   Expected (Smart thermostat)        Expected(Smart light)  \
0                         48.0                           72   
1                         72.0                          108   
2                         60.0                           90   
3                         32.0                           48   
4                         28.0                           42   
5                          NaN                          NaN   
6                          NaN  Chi square statistic value    

   Chi square (Smart thermostat)  chicsqare (Smart light)  
0                       0.083333                 0.055556  
1                       0.888889                 0.592593  
2                       0.000000                 0.000000  
3                       0.125000                 0.083333  
4                       2.285714                 1.523810  
5                            NaN                      NaN  
6                       5.638228                      NaN

From the above data we can see that the value of chi-sqaure statistic is 5.63(approx). 


__3. Determine the Critical Value:__

Alpha = 0.05,
degree of freedom = 5-1 = 4

using the above values from the Chi- square table we get the critical value as 9.488.

__Conclusion__

Our calculated chi - square statistic is 5.638 and our critical value is 9.488

since 5.638 < 9.488, we fail to reject the null hypothesis (H0) and conclude that there's a significant association between the type of smart home device purchased (Smart Thermostats vs. Smart Lights) and the customer satisfaction level.